library/bignum.c

```diff
@@ -2084,6 +2086,22 @@ static int mpi_miller_rabin( const mbedtls_mpi *X,
           ( i >=  650 ) ?  4 : ( i >=  350 ) ?  8 :
           ( i >=  250 ) ? 12 : ( i >=  150 ) ? 18 : 27 );
 
+    if( MPI_FIPS_COMPLIANCE == fips_flag )
+    {
+        /*
+         * FIPS 186-4, table C2
+         *
+         * The 512 bit case is omitted deliberately, because the rest of the
+         * generation process is not permitted for 1024 bit modulus size.
+         */
+        switch(i)
+        {
+            case 1024: n = 5; break;
+            case 1536: n = 4; break;
+            default: ;
```

Comment:
Maintenance: `default: break;` (we wouldn't want a fallthrough if someone adds a case below `default`)